zerver/lib/domains.py

```diff
@@ -4,7 +4,7 @@
 import re
 
 def validate_domain(domain: str) -> None:
-    if domain is None or len(domain) == 0:
+    if not domain:
```

Comment:
I think the right fix is to just make this function accept `Optional[str]`.

I prefer the style we had before -- it makes clear that both possibilities exist here and should behave the same.

Reply:
Understood;  Updated :)